### PR TITLE
Initialize US with data for E2E tests

### PR DESCRIPTION
**Pull request by bt4R9**
In this PR
- Initializing mock data using data from the UI container

Related PR in UI https://github.com/datalens-tech/datalens-ui/pull/129

### CODE REVIEW

File: scripts/preflight.sh [hunk 1]
```diff
@@ -25,5 +25,12 @@ if [ "$USE_DEMO_DATA" = "1" ]; then
     echo "Finish setting up demo data"
 fi
 
+if [ "$USE_E2E_MOCK_DATA" = "1" ]; then
+    echo "Start setting up e2e data"
+    sh /opt/e2e-data/init.sh
```
Comment (vrozaev) [inline scripts/preflight.sh:30]: Could we add more context here via filename?

The name `init-postgres-connection.sh` might be better than just `init.sh`.

File: scripts/preflight.sh [hunk 2]
```diff
@@ -25,5 +25,12 @@ if [ "$USE_DEMO_DATA" = "1" ]; then
...
+    node /opt/app/dist/server/db/scripts/e2e/init.js
```
Comment (vrozaev) [inline scripts/preflight.sh:31]: Could we add more context here via filename?

The name `init-united-storage-data.js` might be better than just `init.js`.

File: src/db/scripts/e2e/init.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+            const sqlData = fs.readFileSync(PATH_TO_DATA, 'utf8').toString().trim();
+            await db.primary.raw(sqlData);
+        }
+
+        process.exit(0);
+    } catch (err) {
+        console.error(err);
+        process.exit(1);
+    }
+})();
```
Comment (vrozaev) [inline src/db/scripts/e2e/init.ts:27]: To discuss: do we really need a separate .ts file to setup united-storage data?

Looks like we can add a few lines to `/opt/e2e-data/init.sh`.

@imsitnikov what do you think?

Comment (bt4R9) [reply src/db/scripts/e2e/init.ts]: It seems like less relevant now as I removed this shell script.

Comment (imsitnikov) [reply src/db/scripts/e2e/init.ts]: The current solution looks good. I don't have any questions.

**Review by vrozaev — APPROVED** [review]

**Pull request merged**